app/forms/support_interface/edit_single_provider_user_form.rb

```diff
@@ -49,7 +49,7 @@ def deselected_provider_permissions
 
     def possible_permissions
       @possible_permissions ||=
-        Provider.where(sync_courses: true).order(:name).map do |provider|
+        Provider.includes(:courses).where(courses: { id: nil }).order(:name).map do |provider|
```

Comment:
Is this not always going to return no permissions? Any saved course will also have an id allocated to it, won't it?